Description: Change the default user the binary will run as to stunnel4
Forwarded: not-needed
Author: Julien Lemoine <speedblue@debian.org>
Author: Luis Rodrigo Gallardo Cruz <rodrigo@debian.org>
Last-Update: 2014-06-10

--- a/tools/stunnel.conf-sample.in
+++ b/tools/stunnel.conf-sample.in
@@ -11,8 +11,8 @@
 ; Chroot conflicts with configuration file reload and many other features
 ;chroot = @localstatedir@/lib/stunnel4/
 ; Chroot jail can be escaped if setuid option is not used
-setuid = nobody
-setgid = @DEFAULT_GROUP@
+setuid = stunnel4
+setgid = stunnel4
 
 ; PID file is created inside the chroot jail (if enabled)
 ;pid = @localstatedir@/run/stunnel4.pid
